py_gtfs_rt_ingestion/dev_test_setup.py

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+import sys
+from typing import NamedTuple
+import os
+import random
+from multiprocessing import Pool
+from collections.abc import Iterable
+
+import boto3
+
+from py_gtfs_rt_ingestion import file_list_from_s3
+
+GTFS_BUCKET = "mbta-gtfs-s3"
+DEV_INGEST_BUCKET = "mbta-ctd-dataplatform-dev-incoming"
+DEV_EXPORT_BUCKET = "mbta-ctd-dataplatform-dev-springboard"
+DEV_ARCHIVE_BUCKET = "mbta-ctd-dataplatform-dev-archive"
+DEV_ERROR_BUCKET = "mbta-ctd-dataplatform-dev-error"
+LAMP_PREFIX = "lamp/"
+
+logging.basicConfig(level=logging.WARNING)
+
+
+class SetupArgs(NamedTuple):
+    """
+    NamedTuple to hold Setup arguments
+    """
+
+    src_prefix: str
+    objs_to_copy: int
+    force_delete: bool
+
+
+def parse_args(args: list[str]) -> SetupArgs:
+    """
+    parse input args from the command line and generate an event dict in the
+    format the lambda handler is expecting
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-s",
+        "--source-prefix",
+        dest="src_prefix",
+        type=str,
+        default="",
+        help="prefix of objects to move into dev environment",
+    )
+
+    parser.add_argument(
+        "-n",
+        "--num",
+        dest="objs_to_copy",
+        type=int,
+        default=1_000,
+        help="number of objects to move into dev environment",
+    )
+
+    parser.add_argument(
+        "-f",
+        dest="force_delete",
+        default=False,
+        action="store_true",
+        help="delete objects from dev buckets without asking",
+    )
```

Comment:
Add argument to delete all objects from dev buckets without asking, should this also apply to the object copy?

Reply:
i would think so.

having a "non user interactive" mode feels like something we would eventually want.